src/App.js

```diff
@@ -1,19 +1,23 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Switch, Route } from 'react-router-dom';
+import Header from 'components/header';
+import Home from 'components/home';
+import Favorites from 'components/favorites';
+import { Provider } from 'react-redux';
+import store from 'store';
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
+      <Provider store={store}>
+        <div className="App">
```

Comment:
you don't need classNames if you use styled-components

Reply:
I thought this one was related to how React handles the root, but removed it and still works, cool! Thanks.